core/src/main/java/com/netflix/iceberg/util/BinPacking.java

```diff
@@ -55,20 +59,22 @@ public ListPacker(long targetWeight, int lookback) {
     private final long targetWeight;
     private final int lookback;
     private final Function<T, Long> weightFunc;
+    private final boolean largestBinFirst;
 
     public PackingIterable(Iterable<T> iterable, long targetWeight, int lookback,
-                           Function<T, Long> weightFunc) {
+                           Function<T, Long> weightFunc, boolean largestBinFirst) {
```

Comment:
Minor: this doesn't seem to follow standard indentation, the line should wrap to 2 spaces forward from previous indentation.